utils/node-loader/src/main.rs

```diff
@@ -61,5 +61,13 @@ async fn load_node(params: LoadParams) -> Result<()> {
         env!("CARGO_PKG_VERSION"),
     );
 
-    BatchPool::<SmallRng>::run(params, rx).await
+    let api = GearApiFacade::try_new(params.node.clone(), params.user.clone()).await?;
+
+    let batch_pool = BatchPool::<SmallRng>::new(
+        api.clone(),
+        params.batch_size,
+        params.workers,
+        rx.resubscribe(),
+    );
+    tokio::spawn(batch_pool.run(params, rx)).await?
```

Comment:
Here we handle workers of the `batch_pool`, but what about the `listen_events`? That thread also returns an error, but it's not handled, as I see

Reply:
Done